crates/control/src/models/builds.rs

```diff
@@ -0,0 +1,49 @@
+use chrono::{DateTime, Utc};
+use models;
+use serde::{Deserialize, Serialize};
+use sqlx::{types::Json, FromRow};
+use std::fmt;
+
+use crate::models::accounts;
+use crate::models::id::Id;
```

Comment:
_This isn't specifically about this file, but I think it's a good example of what I'm talking about below. Nothing needs to change in this PR, I'm just gathering your thoughts._

I'm definitely feeling a naming clash between the `models` crate and the `control::models` module. I think it may be best to rename `control::models` to some other name. I think this would help with conceptually trying to map _which_ `models` module something is referring to. 

I'd originally used `models` simply as pulling from the Rails lingo for this. Phoenix uses `schemas` to talk refer to structs which map to database entities.

What do you think? `control::schemas`? `control::resources`? Other?